composer.json

```diff
@@ -75,7 +75,7 @@
     },
     "require-dev": {
         "aws/aws-sdk-php": "~3.0",
-        "iron-io/iron_mq": "~2.0",
+        "iron-io/iron_mq": "~2.0 || ~4.0",
```

Comment:
Are you referring to the space between the `||` or that there should only be a single `|` I am really surprised the single `|` works :) The documentation for composer says to use double pipe https://getcomposer.org/doc/articles/versions.md#range

(updating to single pipe but just curious your feedback)